### DISCUSSION TITLE
Kernel (SysRq) support.

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [manna-harbour/miryoku#259] [Q&A] [answered]
[Kernel (SysRq)](https://wiki.archlinux.org/title/keyboard_shortcuts#Kernel_(SysRq)), is there a way to perform this? i looked at reference manual but couldn't find any thing.

**Comment by manna-harbour** [comment, marked as answer]
`SysRq` is `Alt`+`PrtScn`. `PrtScn` is on [Fun](https://github.com/manna-harbour/miryoku/tree/master/docs/reference#fun), labelled in the diagrams as `PrtScn SysRq`.

Also, for combinations with `SysRq`, `Alt`+`PrtScn` both don't need to be held when hitting the other key. It's sufficient to hold `Alt`, tap `PrtScn`, tap the other keys, then release `Alt`.

If you want to enter a sequence such as `REISUB` and there is a conflict with the held `Alt`, try `Right Alt` / `AltGr`, available directly below `Alt` on both sides.

**Reply by KAGEYAM4** [reply]
Thank you. I didn't knew about these things.


> Right Alt / AltGr, available directly below Alt on both sides.
was this mentioned in reference manual?

**Reply by sunaku** [reply]
Thanks for the tip!  RALT does appear to work better than LALT: there aren't as many Alt/FileMenu triggers (possibly added by the Desktop Environment I'm using) when observing both variants under `xev`.  For example, [here is my ZMK macro using RALT](https://github.com/sunaku/glove80-keymaps/commit/5d8544df672d5d57e9934c7dcffc220d7f2f2c62).  Cheers.
